content/en/docs/usage/ingress.md

```diff
@@ -85,6 +85,11 @@ trigger `Certificate` resources to be automatically created:
   returned.  This is useful for keeping compatibility with the `ingress-gce`
   component.
 
+## Generate multiple certificates with multiple ingresses
+
+If you need to generate certificates from multiple ingresses make sure it has the issuer annotation.
+Besides the annotation is necessary that each ingress posess a unique `tls.secretName`
```

Comment:
```suggestion
If you need to generate certificates from multiple ingresses make sure it has the issuer annotation.
Besides the annotation, it is necessary that each ingress possess a unique `tls.secretName`.
```

Reply:
I applied these suggestion & rebased this PR.

Reply:
Woah, 2021! 😅🤯

![Screenshot 2023-09-05 at 10 55 47](https://github.com/cert-manager/website/assets/2195781/70979c9b-3542-40c0-9148-7c37c0fdf4f9)